README.md

```diff
@@ -1,32 +1,23 @@
-<!-- prettier-ignore-start -->
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-<img alt="All Contributors: 5 👪" src="https://img.shields.io/badge/all_contributors-5_👪-21bb42.svg" />
+	<!-- prettier-ignore-start -->
+	<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+	<a href="#contributors" target="_blank"><img alt="👪 All Contributors: 5" src="https://img.shields.io/badge/%F0%9F%91%AA_all_contributors-5-21bb42.svg" /></a>
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-<!-- prettier-ignore-end -->
-	</a>
-	<a href="https://codecov.io/gh/JoshuaKGoldberg/tidelift-me-up" target="_blank"><img alt="Codecov Test Coverage" src="https://codecov.io/gh/JoshuaKGoldberg/tidelift-me-up/branch/main/graph/badge.svg"/></a>
-	<a href="https://github.com/JoshuaKGoldberg/tidelift-me-up/blob/main/.github/CODE_OF_CONDUCT.md" target="_blank"><img alt="Contributor Covenant" src="https://img.shields.io/badge/code_of_conduct-enforced-21bb42" /></a>
-	<a href="https://github.com/JoshuaKGoldberg/tidelift-me-up/blob/main/LICENSE.md" target="_blank"><img alt="License: MIT" src="https://img.shields.io/github/license/JoshuaKGoldberg/tidelift-me-up?color=21bb42"></a>
-	<img alt="TypeScript: Strict" src="https://img.shields.io/badge/typescript-strict-21bb42.svg" />
-	<img alt="npm package version" src="https://img.shields.io/npm/v/tidelift-me-up?color=21bb42" />
+	<!-- prettier-ignore-end -->
+	<a href="https://github.com/JoshuaKGoldberg/tidelift-me-up/blob/main/.github/CODE_OF_CONDUCT.md" target="_blank"><img alt="🤝 Code of Conduct: Kept" src="https://img.shields.io/badge/%F0%9F%A4%9D_code_of_conduct-kept-21bb42" /></a>
+	<a href="https://codecov.io/gh/JoshuaKGoldberg/tidelift-me-up" target="_blank"><img alt="🧪 Coverage" src="https://img.shields.io/codecov/c/github/JoshuaKGoldberg/tidelift-me-up?label=%F0%9F%A7%AA%20coverage" /></a>
+	<a href="https://github.com/JoshuaKGoldberg/tidelift-me-up/blob/main/LICENSE.md" target="_blank"><img alt="📝 License: MIT" src="https://img.shields.io/badge/%F0%9F%93%9D_license-MIT-21bb42.svg" /></a>
+	<a href="http://npmjs.com/package/tidelift-me-up" target="_blank"><img alt="📦 npm version" src="https://img.shields.io/npm/v/tidelift-me-up?color=21bb42&label=%F0%9F%93%A6%20npm" /></a>
+	<img alt="💪 TypeScript: Strict" src="https://img.shields.io/badge/%F0%9F%92%AA_typescript-strict-21bb42.svg" />
 </p>
 
-## What?
-
-[Tidelift](https://tidelift.com) is a service for companies to sponsor their open source dependencies.
-Through Tidelift, companies contribute to a fund that is distributed monthly to maintainers of eligible open-source packages in a process called "lifting".
-
-Unfortunately, there's no easy way on the Tidelift website to search for which packages under a particular open source maintainer are eligible for lifting or currently being lifted.
-This CLI does that!
-
-> See it in action online at [tidelift-me-up-site.vercel.app](https://tidelift-me-up-site.vercel.app)!
-
```

Comment:
🐛 Agh, I meant to revert this change before merging. CTA shouldn't have removed it.

Reply:
https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2127